README.rst

```diff
@@ -95,7 +95,7 @@ Listed below are example Jupyter notebooks for working with ICESat-2 (IS2).
 
 `IS2_data_read-in <https://icepyx.readthedocs.io/en/latest/example_notebooks/IS2_data_read-in.html>`_
 
-`IS2_cloud_data_access (BETA ONLY) <https://icepyx.readthedocs.io/en/latest/example_notebooks/IS2_cloud_data_access.html>`_
+`IS2_cloud_data_access <https://icepyx.readthedocs.io/en/latest/example_notebooks/IS2_cloud_data_access.html>`_
```

Comment:
For this (and the other examples), maybe we should specify what each example outlines. It's obvious for the cloud access notebook, though users might not know what "data_variables" or "data_read-in" refers to.

Reply:
Agreed - I'm going to open an issue to add an examples index page of some sort to provide this info (and then we can include it or link to it from the readme).